src/EnergyPlus/DataSurfaces.hh

```diff
@@ -606,6 +606,9 @@ namespace DataSurfaces {
 		Real64 Width; // Width of the surface (m)
 		// Boundary conditions and interconnections
 		bool HeatTransSurf; // True if surface is a heat transfer surface,
+		bool HasAdditionalHeatSourceTerm; // True if surface has additional heat flux source as boundary condition
+		bool OutsideAddHeatSourceTerm; // True if the additional heat flux source applies to the exterior side of the surface, and false if applies to interior side
```

Comment:
This is not the most straightforward variable naming scheme.  Saying False to indicate that there is still some impact, just on the other side, is confusing to debug.  And what if the user applies a source to both sides??

How about instead of HasAdditionalHeatSourceTerm and OutsideAddHeatSourceTerm, you just use: HasOutsideHeatSourceTerm and HasInsideHeatSourceTerm.

And on the schedule variable here, if the user applies a source term to both the inside and outside, do you also need to break that into: OutsideHeatSourceTermSchedule and InsideHeatSourceTermSchedule?

Reply:
We updated the variables to have only OutsideAddHeatSourceTermSch and InsideAddHeatSourceTermSch.